src/ai-bolit.php

```diff
@@ -2433,6 +2440,7 @@ function QCR_ScanFile($l_Filename, $i = 0)
 
 			$l_TSStartScan = microtime(true);
                 $l_Content = @file_get_contents($l_Filename);
+                $l_Unwrapped = @php_strip_whitespace($l_Filename);

```

Comment:
Shall we put this line 4 lines below? Right after if (($l_Content == '') && ($l_Stat['size'] > 0)) { ... } ?